test/local_integration_test.py

```diff
@@ -86,6 +89,15 @@ def test_webservice_and_indexer(self):
                     response = self.check_endpoint_is_working(config.service_endpoint(), path, query)
                     validator(response)
 
+        filters = {"file":{"organ":{"is":["Brain"]},"fileFormat":{"is":["fastq"]}}}
```

Comment:
```suggestion
        filters = {"file":{"organ":{"is":["brain"]},"fileFormat":{"is":["fastq"]}}}
```

Reply:
Good catch - changed. Thanks.

Reply:
After changing organ to `brain` I needed to change `fastq` to `fastq.gz` in addition. If organ is `Brain`, both file formats are available, for `brain` only `fastq.gz`, not `fastq` is in the index.